tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Repository/Configs/ConfigRepository.cs

```diff
@@ -82,7 +82,10 @@ public void UploadConfigFileToSPO(string filePath, string clientUrl)
                 clientContext.Load(uploadFile);
                 clientContext.ExecuteQuery();
                 //After uploading the file to sharepoint site collection, delete the file from the app root
-                System.IO.File.Delete(filePath);
+                if(!filePath.EndsWith("config.js"))

```

Comment:
@WilsonReddy If the file is config.js, we are not deleting that file as that file will be used by matter center also